crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -41,6 +43,7 @@ pub enum CipherError {
 #[derive(Clone, Copy, Serialize_repr, Deserialize_repr, Debug, JsonSchema, PartialEq)]
 #[repr(u8)]
 #[cfg_attr(feature = "uniffi", derive(uniffi::Enum))]
+#[wasm_bindgen]
```

Comment:
This should only be derived when the wasm target is enabled, same for `CipherRepromptType`, `FieldType`, `LoginLinkedIdType`, etc.
```suggestion
#[cfg_attr(feature = "wasm", wasm_bindgen)]
```